python_ta/cfg/visitor.py

```diff
@@ -235,7 +235,11 @@ def _visit_jump(
     ) -> None:
         old_curr = self._current_block
         for boundary, exits in reversed(self._control_boundaries):
-            if type(node).__name__ in exits:
+            if isinstance(node, nodes.Raise) and f"{nodes.Raise.__name__} {node.exc.name}" in exits:
```

Comment:
It's important to be careful when going deeper into AST structure. In particular, we shouldn't assume `node.exc` is a `Name` node or has a `name` attribute. Consider

```python
raise ValueError

raise ValueError(10)
```

This might require a helper function to be more general (and I think this can just default to `'Raise'` if the `exc` is of an unexpected type).